### ISSUE TITLE
Make shell-randomquote POSIX-compliant

### ISSUE THREAD

**Issue by juanluisvaladas** [fbac/shell-randomquote#1]
Shuf is not POSIX compliant and it's not shipped by default on most of linux and BSD distributions and even if it is a coreutil it's rarely shipped in such pakages.

**Comment by jparrill** [comment]
#4 Merged ;)

**Comment by juanluisvaladas** [comment]
@padajuan not yet. Even after #5 RANDOM is not POSIX compliant T_T

**Comment by juanluisvaladas** [comment]
WONTFIX
